src/main/java/de/zalando/zmon/dataservice/data/KairosDBStore.java

```diff
@@ -251,6 +253,10 @@ void store(WorkerResult wr) {
                     LOG.warn("result size warning: check={} data-points={} job-related={} entity={} tags={}", cd.checkId, cdResultSize, isJobRelated, cd.entityId, getEntityTags(cd.entity));
                 }
 
+                if (cdResultSize > resultSizeMetricThreshold) {
+                    metrics.markCriticalCheck(cd.checkId, wr.account, cdResultSize);
```

Comment:
Not really sure about `account` inside of the `WorkerResult` here.  Need to check if it's the same as `accountId` on the API handler, or find a way to pass it though.

Reply:
this is the accound id from the URL